client/src/helper-functions/rnRHelper.js

```diff
@@ -114,8 +114,22 @@ const helper = {
       return starFilters.includes(review.rating);
     })
   },
-  auditReviews: (review) => {
-    return true;
+  validateEmail: (email) => {
+    const regex = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
```

Comment:
Is this a regex you were able to find online for email validation? Seems like a crazy thing to have to come up with if not